spec/factorial_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rspec'
+require_relative '../classes/factorial'
+
+describe Factorial do
+  it 'should return the factorial of a given number' do
+    calc = Factorial.new
+    expect(calc.find(4)).to eq(24)
+  end
+  it 'should return 1 for a factorial of 0' do
+    calc = Factorial.new
+    expect(calc.find(0)).to eq(1)
+  end
+end
```

Comment:
- Great Job for testing factorial o and positive number, I think it would be better if you raise an exception if negative values passed as mentioned in the requirement 👍
![Screenshot 2023-06-29 141704](https://github.com/joyapisi/tdd_ruby/assets/52010789/f5020d6e-89b0-4f6c-848a-a35ae2e3e962)

check the bellow snippet 👇
```
     describe Resolver do
         describe "#factorial" do
             it "raises exception for negative number" do
             resolver = Resolver.new
             expect { resolver.factorial(-5) }.to raise_exception
             end
         end
     end
```